src/lib/aggregations/get-bucket-sort.ts

```diff
@@ -0,0 +1,22 @@
+import { Order } from 'lib/enums'
+
+export type BucketSortAggregation = {
+    bucket_sort: BucketSortAggregationBody
+}
+
+export type BucketSort = {
+    [bucketPath: string]: { order: Order }
+}
+
+// note: all field are optional, because
+// you can use bucket sort for both sorting and truncating result buckets
+// but you also can use it for only sorting or only truncating
```

Comment:
maybe `@description` instead?

Reply:
alright

Reply:
in that case the `parameters` should be also optional 🙏 (defaulting to empty `{}` if undefined)